features/scripting.yml

```diff
@@ -0,0 +1,5 @@
+name: scripting media query
+description: The `scripting` CSS media query sets styles based on whether scripting such as JavaScript is available. Values are `enabled` if scripting is available, `initial-only` if scripting is only available on page load (for example, printed or server-rendered content), or `none`.
```

Comment:
The server-rendered bit has me scratching my head a little. Is this a real thing that people do?

Reply:
This is a use case from the spec, and an example would be Opera Mini, which prerenders on a server, and then has limited JS functionality afterwards. But it does seem like a less practical example, so I removed it.